25_pair_code_functions.h

```diff
@@ -0,0 +1,31 @@
+//File contains enums, structure declarations and function protoypes to implement 25_pair_code_functions
```

Comment:
File naming can be improved. Right now, there are 25 colors. Hence file naming seems ok. But if there is an increase or decrease in the color number, then this name shall be irrelevant !
This is about color coding, so names should be chosen appropriately